tests/src/whisk/core/controller/test/ActionsApiTests.scala

```diff
@@ -239,55 +237,6 @@ class ActionsApiTests extends ControllerTestCommon with WhiskActionsApi {
         }
     }
 
```

Comment:
the tests do belong here - otherwise you're not really testing the api.
the unit test for the directive can be its own separate test.